src/main/java/Duke.java

```diff
@@ -1,10 +1,177 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    private static final String horizontalBorder = "_________________________________\n";
+    private static final String INVALID_TODO_INPUT = " ☹ OOPS!!! The description of a todo cannot be empty.\n";
+    private static final String INVALID_DEADLINE_INPUT = "☹ OOPS!!! Please use proper deadline formatting: deadline {task} /by {time}\n";
+    private static final String INVALID_EVENT_INPUT = "☹ OOPS!!! Please use proper event formatting: event {task} /at {time}\n";
+    private static final String INVALID_ACCESS_EMPTY_TASKLIST = "☹ OOPS!!! Task does not exist. Initialise a task first, then try again\n";
+    private static final String INVALID_USER_INPUT = "☹ OOPS!!! Please use one of these keywords: {deadline, event, todo} followed by \\\"by\\\" and \\\"at\\\" for deadline and event tasks respectively.\n";
+
+    private Tasklist tasklist;
+
+    private Duke(){
```

Comment:
It does not appear that this constructor is invoked so you could look into omitting this constructor.